src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -4441,8 +4441,13 @@ checkNullable:
                     ' being reported. For now keep backwards compatibility.
                     If param.ContainsDiagnostics Then
                         param = param.AddTrailingSyntax(ResyncAt({SyntaxKind.CommaToken, SyntaxKind.CloseParenToken}))
-                    End If
+                    Else
+                        ' TODO: MAKE THIS CONDITIONAL ON A LANGUAGE FEATURE FLAG
+                        If param.Default Is Nothing Then
```

Comment:
It feels like this (reporting an error based on the target language version) should be done in ParseParameter